src/main/java/org/eolang/speco/Main.java

```diff
@@ -61,9 +61,24 @@ public final class Main implements Callable<Integer> {
         description = "If the input program is in EO")
     private boolean eolang;
 
+    /**
+     * Flag indicating whether the temporary tags should be deleted.
+     */
+    @CommandLine.Option(names = { "--clear-xmir" },
+        defaultValue = "false",
+        description = "If delete temporary tags")
+    private boolean clearxmir;
+
     @Override
     public Integer call() throws IOException {
-        new Speco(this.input, this.output, this.eolang).exec();
+        Speco speco = new DefaultSpeco(this.input, this.output);
+        if (this.eolang) {
+            speco = new EolangSpeco(speco);
```

Comment:
@volodya-lombrozo I don't think utility class is a good idea -- https://www.yegor256.com/2014/05/05/oop-alternative-to-utility-classes.html#utility-classes-are-evil :)

Reply:
@volodya-lombrozo btw, the current design allows for changes in multiple dimensions (input/output format (`EolangSpeco`) and flags (`ClearXmirSpeco`), which can be combined. That is, it may become a problem in the future when new `*Speco` is added